test/recipes/95-test_external_tlsfuzzer_data/cert.json.in

```diff
@@ -12,12 +12,12 @@
        {"name" : "test-tls13-certificate-verify.py",
         "arguments" : ["-k", "tests/clientX509Key.pem",
                        "-c", "tests/clientX509Cert.pem",
-                       "-s", "9+5 9+6 9+4 ecdsa_secp256r1_sha256 ecdsa_secp384r1_sha384 ecdsa_secp521r1_sha512 ed25519 ed448 8+26 8+27 8+28 rsa_pss_pss_sha256 rsa_pss_pss_sha384 rsa_pss_pss_sha512 rsa_pss_rsae_sha256 rsa_pss_rsae_sha384 rsa_pss_rsae_sha512 rsa_pkcs1_sha256 rsa_pkcs1_sha384 rsa_pkcs1_sha512 ecdsa_sha224 rsa_pkcs1_sha224",
+                       "-s", "9+5 9+6 9+4 ecdsa_secp256r1_sha256 ecdsa_secp384r1_sha384 ecdsa_secp521r1_sha512 ed25519 ed448 8+26 8+27 8+28 rsa_pss_pss_sha256 rsa_pss_pss_sha384 rsa_pss_pss_sha512 rsa_pss_rsae_sha256 rsa_pss_rsae_sha384 rsa_pss_rsae_sha512",
```

Comment:
So we aren't sending the RSA PKCS#1 sigalgs by default anymore? What if the peer is strictly checking that? Or do I misunderstand?

Reply:
We don't send in a client certificate request once TLS 1.3 is negotiated, because TLS 1.2 compatibility is no longer required, and these sigalgs are for TLS 1.2 only.

Reply:
They're of course sent by clients to the server, when we're version-flexible (willing to negotiate at least either of TLS 1.2 or TLS 1.3).